src/Parser/LASParser.js

```diff
@@ -1,66 +1,76 @@
 import * as THREE from 'three';
-import { LASLoader } from '@loaders.gl/las';
+import LASLoader from 'Parser/LASLoader';
 
-// See this document for LAS format specification
-// https://www.asprs.org/wp-content/uploads/2010/12/LAS_1_4_r13.pdf
-// http://www.cs.unc.edu/~isenburg/lastools/download/laszip.pdf
+const lasLoader = new LASLoader();
 
-/**
- * The LASParser module provides a [parse]{@link module:LASParser.parse} method
- * that takes a LAS file or a LAZ (LASZip) file in, and gives a
- * `THREE.BufferGeometry` containing all the necessary attributes to be
+/** The LASParser module provides a [parse]{@link
+ * module:LASParser.parse} method that takes a LAS or LAZ (LASZip) file in, and
+ * gives a `THREE.BufferGeometry` containing all the necessary attributes to be
  * displayed in iTowns. It uses the
- * [LASLoader](https://loaders.gl/modules/las/docs/api-reference/las-loader)
- * from `loaders.gl`.
+ * [copc.js](https://github.com/connormanning/copc.js/) library.
  *
  * @module LASParser
  */
 export default {
+    /*
+     * Set the laz-perf decoder path.
+     * @param {string} path - path to `laz-perf.wasm` folder.
+     */
+    enableLazPerf(path) {
+        if (!path) {
+            throw new Error('Path to laz-perf is mandatory');
```

Comment:
Is it? There is a default in LasLoader. Should we remove the default and indicate to the user that one can use a cdn ?

Reply:
We followed the naming convention of the `enable*` family of functions of 3DTiles parsers. Since removing the default cause a breaking changes for users of `EntwinePointTileLayer`, I advocate for keeping a default CDN.

Reply:
fine by me